src/DocumentDB.ChangeFeedProcessor.IntegrationTests/SmokeTest.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
```

Comment:
consider moving all usings inside namespace, then we won't need explicit "FeedProcessing.".

Reply:
I tried but 
1) I still need them just Microsoft.Azure.Documents.ChangeFeedProcessor will be removed
2) it will hide when the namespaces comes from
3) it has different semantic than outside of namespace (I dont remember exact usacase now but we had an issue with class/interface resolution order)